mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/quota/CassandraQuotaCurrentValueDao.java

```diff
@@ -125,15 +129,27 @@ public Mono<Void> increase(QuotaKey quotaKey, long amount) {
             .setString(QUOTA_COMPONENT, quotaKey.getQuotaComponent().getValue())
             .setString(IDENTIFIER, quotaKey.getIdentifier().asString())
             .setString(QUOTA_TYPE, quotaKey.getQuotaType().getValue())
-            .setLong(CURRENT_VALUE, amount));
+            .setLong(CURRENT_VALUE, amount))
+            .onErrorResume(ex -> {
+                LOGGER.error("Failure when increasing {} {} quota for {}. Quota current value is thus not updated and needs recomputation",
+                    quotaKey.getQuotaComponent().getValue(), quotaKey.getQuotaType().getValue(), quotaKey.getIdentifier().asString());
+                LOGGER.error(ex.getMessage());
+                return Mono.empty();
```

Comment:
```suggestion
                LOGGER.warn("Failure when increasing {} {} quota for {}. Quota current value is thus not updated and needs recomputation",
                    quotaKey.getQuotaComponent().getValue(), quotaKey.getQuotaType().getValue(), quotaKey.getIdentifier().asString(), ex);
                return Mono.empty();
```

 -> WARN: this won't prevent the system from working, no need to be so worrysome.
 -> Only one line logs or correlating both logs will be impossible
 -> Do not loose stack traces